python/tests/core/test_model_performance_metrics.py

```diff
@@ -273,3 +277,14 @@ def test_profile_top_level_api_segmented_performance():
     assert metrics1.confusion_matrix.labels == ["x0", "x1"]
     assert metrics1.confusion_matrix.confusion_matrix.get((0, 0)).n == 1
     assert metrics1.confusion_matrix.confusion_matrix.get((1, 0)).n == 1
+
+def test_deserialize_confusion_matrix_with_kll_floats() -> None:
+    sample = \
+        'CgEwCgExEhxkZWxpdmVyeV9wcmVkaWN0aW9uIChvdXRwdXQpGhhkZWxpdmVyeV9zdGF0dXMgKG91dHB1dCkiHGRlbGl2ZXJ5X2NvbmZpZGVuY2UgKG91dHB1dClSFgoAIggCAQ8BAAEIADIIAQMDAAAezJNSFgoAIggCAQ8BAAEIADIIAQMDAAAezJNS0QEKFAgKEUcu/yH99rU/GeF6FK5H4eo/Eh0IChGuR+F6FK7nPxkAAAAAAADwPyHNzMzMzMwgQCJIBQEPAAABCAAKAAAAAAAAAAABAQD2AAAApHA9PwAAgD/Xo3A/uB5FP6RwPT/NzEw/PQpXP0jhej+kcD0/cT1KPwAAgD/NzEw/MlACAwMAABrMkwgAAAAAAAAA08wssPU9DQg/rZkmPpX4CJp1jlD5me4aoWIjAG6MNiCLJTyeLc5zUuyIHr4Td6ZphdLO5crXo3zWNEUNX4OGflIWCgAiCAIBDwEAAQgAMggBAwMAAB7Mkw=='
+    bytes = base64.b64decode(sample)
+    msg = ScoreMatrixMessage()
+    msg.ParseFromString(bytes)
+    confusion_matrix = ConfusionMatrix.from_protobuf(msg)
+    assert confusion_matrix is not None
+    merged = confusion_matrix.merge(confusion_matrix)
+    merged.to_protobuf()
```

Comment:
Is there an assert missing, or the test is just that the last two lines don't crash?

Reply:
The second case - want to make sure it doesn't crash